modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/TestFulltextStoredInBlob.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * (C) Copyright 2015-2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Benoit Delbosc
+ *     Florent Guillaume
+ */
+package org.nuxeo.elasticsearch.test;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.DocumentModelList;
+import org.nuxeo.ecm.core.query.QueryParseException;
+import org.nuxeo.ecm.core.test.FulltextStoredInBlobFeature;
+import org.nuxeo.elasticsearch.query.NxQueryBuilder;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+@RunWith(FeaturesRunner.class)
+@Features({ FulltextStoredInBlobFeature.class, RepositoryElasticSearchFeature.class })
+@Deploy("org.nuxeo.elasticsearch.core:elasticsearch-test-contrib.xml")
+public class TestFulltextStoredInBlob extends TestFulltextEnabled {
+
+    @Override
+    @Test
+    public void testFulltext() throws Exception {
+        createFileWithBlob();
+        // binary fulltext extraction is done
+        String nxql = "SELECT * FROM Document WHERE ecm:fulltext='search'";
+        DocumentModelList esRet = ess.query(new NxQueryBuilder(session).nxql(nxql));
+        Assert.assertEquals(1, esRet.totalSize());
```

Comment:
```suggestion
        assertEquals(1, esRet.totalSize());
```
There's another one in the class.

Reply:
This class is actually essentially a copy of an existing one but you're right, I'll clean it up.